multiclusterhub/charts/cluster-lifecycle/templates/clustermanagementaddon-clusterrolebinding.yaml

```diff
@@ -0,0 +1,14 @@
+# Copyright Contributors to the Open Cluster Management project
+
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRoleBinding
+metadata:
+  name: {{ .Values.org }}:{{ .Release.Name }}:clustermanagementaddons-readonly-v2
+subjects:
+- kind: Group
+  name: system:authenticated
+  apiGroup: rbac.authorization.k8s.io
+roleRef:
+  kind: ClusterRole
+  name: {{ .Values.org }}:{{ .Release.Name }}:clustermanagementaddons-readonly-v2
+  apiGroup: rbac.authorization.k8s.io
```

Comment:
Missing newline at end of last line.

https://stackoverflow.com/a/729795/1437822